Examples/DicomSeriesFromArray/DicomSeriesFromArray.py

```diff
@@ -0,0 +1,116 @@
+#=========================================================================
+#
+#  Copyright Insight Software Consortium
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#=========================================================================
+
+from __future__ import print_function
+
+import SimpleITK as sitk
+
+import sys, time, os
+import numpy as np
+
+if len( sys.argv ) < 2:
+    print( "Usage: python " + __file__ + "<output_directory>" )
+    sys.exit ( 1 )
+
+
+# Create a new series from a numpy array
+new_arr = np.random.uniform(-10, 10, size = (3,4,5)).astype(np.int16)
+new_img = sitk.GetImageFromArray(new_arr)
+new_img.SetSpacing([2.5,3.5,4.5])
+
+
+# Write the 3D image as a series
+# IMPORTANT: There are many DICOM tags that need to be updated when you modify an
+#            original image. This is a delicate opration and requires knowlege of
+#            the DICOM standard. This example only modifies some. For a more complete
+#            list of tags that need to be modified see:
+#                           http://gdcm.sourceforge.net/wiki/index.php/Writing_DICOM
+
+writer = sitk.ImageFileWriter()
+# Use the study/series/frame of reference information given in the meta-data
+# dictionary and not the automatically generated information from the file IO
+writer.KeepOriginalImageUIDOn()
+
+# Copy relevant tags from the original meta-data dictionary (private tags are also
+# accessible).
+tags_to_copy = ["0010|0010", # Patient Name
+                "0010|0020", # Patient ID
+                "0010|0030", # Patient Birth Date
+                "0020|000D", # Study Instance UID, for machine consumption
+                "0020|0010", # Study ID, for human consumption
+                "0008|0020", # Study Date
+                "0008|0030", # Study Time
+                "0008|0050", # Accession Number
+                "0008|0060"  # Modality
+]
+
+modification_time = time.strftime("%H%M%S")
+modification_date = time.strftime("%Y%m%d")
+
+# Copy some of the tags and add the relevant tags indicating the change.
+# For the series instance UID (0020|000e), each of the components is a number, cannot start
+# with zero, and separated by a '.' We create a unique series ID using the date and time.
+# tags of interest:
+direction = new_img.GetDirection()
+series_tag_values = [(k, new_img.GetMetaData(k)) for k in tags_to_copy if new_img.HasMetaDataKey(k)] + \
```

Comment:
Please remove the list comprehension, new_img.HasMetaDataKey(k) is always false because the new_img has an empty meta-data dictionary.